homeassistant/components/autarco/sensor.py

```diff
@@ -66,20 +66,58 @@ class AutarcoSolarSensorEntityDescription(SensorEntityDescription):
 )
 
 
+@dataclass(frozen=True, kw_only=True)
+class AutarcoInverterSensorEntityDescription(SensorEntityDescription):
+    """Describes an Autarco inverter sensor entity."""
+
+    state: Callable[[Inverter], StateType]
```

Comment:
```suggestion
    value_fn: Callable[[Inverter], StateType]
```
I personally always prefer value_fn as its a bit more descriptive and it is clear that the field is a function